wdl_runner/README.md

```diff
@@ -0,0 +1,225 @@
+# Run a WDL workflow
+
+## Overview
+
+This example demonstrates running a multi-stage workflow on
+Google Cloud Platform.
+
+* The workflow is launched with the Google Genomics [Pipelines API](https://cloud.google.com/genomics/v1alpha2/pipelines).
+* The workflow is defined using the Broad Institute's
+[Workflow Definition Language](https://software.broadinstitute.org/wdl/) (WDL).
+* The workflow stages are orchestrated by the Broad Institute's
+[Cromwell](https://github.com/broadinstitute/cromwell).
+
+When submitted using the Pipelines API, the workflow runs
+on multiple [Google Compute Engine](https://cloud.google.com/compute/)
+virtual machines.
+First a master node is created for Cromwell, and then Cromwell submits
+each stage of the workflow as one or more separate pipelines.
+
+Execution of a running Pipeline proceeds as:
+
+1. Create Compute Engine virtual machine
+
+2. Execute wdl_runner.py
+    a. Run Cromwell (server)
```

Comment:
Might need a space between these two lines as they appear on the same line after being reformatted.